tests/Util/ApiTestDatasetTrait.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Friendica\Test\Util;
+
+/**
+ * This trait contains the datasets for each ApiTest
+ */
+trait ApiTestDatasetTrait
+{
+	public function dataDefaultUser()
```

Comment:
This one is pretty nice.
https://phpunit.de/manual/3.7/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers.examples.DataTest.php
We can define different datasets and automatically pass them iterative to the test-function. I think we can have here a lot of possibilities (f.e. default-dataset, "invalid"-datasets, "incomplete"-datasets).

The last one is maybe good to find the PHP notices :-)